src/io.jl

```diff
@@ -48,8 +48,15 @@ function read(fn::Union{AbstractString,Parquet2.FilePathsBase.AbstractPath,Parqu
     is_valid(ds) || error("Not a valid GeoParquet file")
     meta = geometadata(ds)
     df = DataFrame(ds; copycols=false)
-    for column in keys(meta.columns)
+    geocolumns = Symbol.(keys(meta.columns))
+    for column in geocolumns
         df[!, column] = GFT.WellKnownBinary.(Ref(GFT.Geom()), df[!, column])
     end
+    # set GeoInterface metadata
+    metadata!(df, "GEOINTERFACE:geometrycolumns", geocolumns)
```

Comment:
I'm not sure if we have clarified if this is a tuple or vector in metadata....

Reply:
I think that can be a tuple